src/api/entities/SecurityToken/index.ts

```diff
@@ -320,12 +320,12 @@ export class SecurityToken extends Entity<UniqueIdentifiers, string> {
 
     if (callback) {
       return asset.fundingRound(rawTicker, round => {
-        callback(fundingRoundNameToString(round));
+        callback(fundingRoundNameToString(round) || null);
       });
     }
 
     const fundingRound = await asset.fundingRound(rawTicker);
-    return fundingRoundNameToString(fundingRound);
+    return fundingRoundNameToString(fundingRound) || null;
```

Comment:
This behavior should probably be encapsulated into a function

```
const assembleResult = (roundName: FundingRoundName): string | null => fundingRoundNameToString(roundName) || null;

if (callback) {
  return asset.fundingRound(rawTicker, round => {
    callback(assembleResult(round));
  });
}

const fundingRound = await asset.fundingRound(rawTicker);
return assembleResult(fundingRound);
```

Easier to change when we move to Options, less prone to bugs

Reply:
I infact did that and then reverted it 😅